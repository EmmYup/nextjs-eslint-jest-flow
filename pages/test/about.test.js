import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

import App from '../about';

describe('With Enzyme', () => {
  it('App shows "About"', () => {
    const app = shallow(<App />);

    expect(app.find('p').text()).toEqual('About');
  });
});

describe('With Snapshot Testing', () => {
  it('App shows "About"', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
