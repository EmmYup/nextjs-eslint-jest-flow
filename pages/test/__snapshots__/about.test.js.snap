// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "About" 1`] = `
<div>
  <header>
    <nav>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      |
      <a
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
    </nav>
  </header>
  <div>
    <p>
      About
    </p>
     
  </div>
  <footer>
    I\`m here to stay
  </footer>
</div>
`;
